ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -289,16 +295,22 @@ extension PopupMenuController: UITableViewDataSource {
         let row = indexPath.row
         let item = sections[section].items[row]
 
-        let cell = tableView.dequeueReusableCell(withIdentifier: PopupMenuItemCell.identifier) as! PopupMenuItemCell
+        let cellClass = item.cellClass
+        let identifier = String(describing: cellClass)
+        self.tableView.register(cellClass, forCellReuseIdentifier: identifier)
+
+        let cell = tableView.dequeueReusableCell(withIdentifier: identifier) as! PopupMenuItemTemplateCell
         cell.setup(item: item)
         cell.preservesSpaceForImage = itemsHaveImages
+
         let isLastInSection = row == tableView.numberOfRows(inSection: section) - 1
         if section == tableView.numberOfSections - 1 && isLastInSection {
-            cell.bottomSeparatorType = .none
+            cell.expectedSeparatorType = .none
         } else {
-            cell.bottomSeparatorType = isLastInSection ? .full : .inset
-            cell.bottomSeparator.backgroundColor = separatorColor
+            cell.expectedSeparatorColor = self.separatorColor
```

Comment:
There's an interesting thing that the property name can't be separator color that system will assign it to default value after layoutIfNeede(), I guess UITableViewCell has a private property with the same name.
https://github.com/modocache/UIKit-Class-Dump/blob/master/UIKit.framework/iPhoneSimulator.platform/UITableViewCell.h
@harrieshin